kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/base/BaseOperation.java

```diff
@@ -824,12 +817,51 @@ protected T handlePatch(T current, T updated) throws ExecutionException, Interru
     return handlePatch(current, updated, getType());
   }
 
+  protected Scale handleScale() {
+    try {
```

Comment:
can we replace the code block by `return handleScale(null)` so we don't have duplicate code?